module/TaskManagement/src/TaskManagement/Controller/TasksController.php

```diff
@@ -155,7 +155,12 @@ public function create($data)
 
 		$this->transaction()->begin();
 		try {
-			$task = Task::create($stream, $subject, $this->identity());
+			if(isset($data['status'])){
+				$options = array('status'=>$data['status']);

```

Comment:
Non permettiamo ad un client di decidere lo stato di un'entità, perché lo stato di un'entità e la consistenza dei dati che ne è legata deve essere una responsabilità del server.
Crea un task in stato Work Item Idea e basta. Mettendo la funzionalità di "start" come unica possibile. La funzionalità di start è un palliativo che porta la work item idea in ongoing (chiamando l'API `transition` (in realtà questa funzionalità verrà utilizzata in futuro sulle Open Work Item)